test/deploy/kubernetes/runtime-pods/components_ha_node.yaml

```diff
@@ -20,7 +20,9 @@ spec:
     metadata:
       labels:
         app: ha-node
+        cortx.io/machine-id: 1115f539f4f770e2a3fe9e2e615c32a8
```

Comment:
Can we pull this value directly from pod metadata id? Instead of explicitly specifying it